R/get_ltcs_path.R

```diff
@@ -12,7 +12,7 @@
 get_ltcs_path <- function(year, ...) {
   ltcs_file_path <- get_file_path(
     directory = fs::path(get_slf_dir(), "LTCs"),
-    file_name = glue::glue("LTC_patient_reference_file-20{year}.rds"),
+    file_name = glue::glue("LTCs_patient_reference_file-20{year}.zsav"),
```

Comment:
I think this needs to stay as `.rds`. However, if there is no `.rds` file available in the folder, you can either run this (im sure the ltc extract has been converted) or alternatively when reading the file you can specify `ext = .zsav`.

Reply:
Changed this back to what it was originally